pyproject.toml

```diff
@@ -53,66 +84,3 @@ addopts = [
 markers = [
     "extended: mark test to be part of the extended test suite"
 ]
-filterwarnings =[
-    "ignore::FutureWarning:rpy2",
-    "ignore::DeprecationWarning:rpy2"
-]
-
-[tool.ruff]
-line-length = 88
-fix = true
-
-# Assume Python 3.9
-target-version = "py39"
-
-[tool.ruff.lint]
-# docs: https://docs.astral.sh/ruff/rules/
-select = [
-  "F", # Pyflakes
-  "E", # pycodestyle errors
-  "W", # pycodestyle warnings
-  "I", # isort
-  "D", # flake8-docstrings
-  "UP", # pyupgrade
-  "SIM", # flake8-simplify
-  "TRY", # tryceratops
-]
-
-ignore = [
-# do not enable if formatting
-# docs: https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules
-  "W191",   # tab indentation
-  "E111",   # indentation
-  "E114",   # indentation
-  "E117",   # over indented
-  "D206",   # indent with spaces
-  "D300",   # triple single quotes
-  "E501",   # line length regulated by formatter
-  "D105",   # missing docstring in magic method
-  "D100",   # missing docstring in public module
-  "D104",   # missing docstring in public package
-  "SIM110", # Use all instead of `for` loop
-  "TRY003", # Avoid specifying long messages outside the exception class
-  "D205",   # 1 blank line required between summary line and description
-  "W505",   # Doc line too long
-]
-
-[tool.ruff.lint.per-file-ignores]
-"tests/**/*.py" = ["D100", "D103"]
-"pyfixest/multcomp.py" = ["D103"]
-"pyfixest/summarize.py" = ["D103"]
-"pyfixest/visualize.py" = ["D103"]
-
-[tool.ruff.lint.pycodestyle]
-max-doc-length = 88
-
-[tool.ruff.lint.pydocstyle]
-convention = "numpy"
```

Comment:
We should not delete the ruff config, right?

Reply:
Yes, I'll bring all of this back, forgot to mention this on top of the PR 😅